src/NuGet.Core/NuGet.ProjectModel/DependencyGraphSpec.cs

```diff
@@ -243,56 +226,55 @@ public static DependencyGraphSpec Union(IEnumerable<DependencyGraphSpec> dgSpecs
 
         public static DependencyGraphSpec Load(string path)
         {
-            using (var stream = new FileStream(path, FileMode.Open, FileAccess.Read, FileShare.Read))
-            using (var streamReader = new StreamReader(stream))
-            using (var jsonReader = new JsonTextReader(streamReader))
-            {
-                var dgspec = new DependencyGraphSpec();
-                bool wasObjectRead;
+            using var stream = new FileStream(path, FileMode.Open, FileAccess.Read, FileShare.Read);
```

Comment:
Given that this is a suggestion and syntactic sugar, I personally don't think we should too aggressive in makes changes like this since they heavily affected the `blame-ability` of a file. 

I'm not asking you to revert it, but in general, I think we should be less aggressive with making large changes

Reply:
In general, I'm more lenient than Nikolche about code cleanup, but the first commit in this branch had 7 lines changed, whereas the second had 125 lines changed.  This makes it difficult to see what the actual product changes are. But thank you for doing it in separate commits. It helped me a lot in reviewing this PR.